TrailblazerIOS/API/UserAPI.swift

```diff
@@ -91,10 +98,10 @@ extension APIHandler {
         URLSession.shared.dataTask(with: request) { data, response, error in
```

Comment:
I think this entire function should be a completion with a result of either a User or an Error, so that its the same as the rest of the functions in the file. So this would require you to make the same changes as the other methods.